google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/admin/v2/stub/BigtableTableAdminStubSettings.java

```diff
@@ -841,7 +839,7 @@ private static Builder initDefaults(Builder builder) {
 
       builder
           .getIamPolicySettings()
-          .setRetryableCodes(RETRYABLE_CODE_DEFINITIONS.get("idempotent"))
+          .setRetryableCodes(RETRYABLE_CODE_DEFINITIONS.get("non_idempotent"))
```

Comment:
This doesn't look right

Reply:
good catch - thank you!

Reply:
This has been fixed upstream. Closing this out to regen.